test/specs/lib.davIdIcon.spec.js

```diff
@@ -17,4 +17,18 @@ describe('getDavIdIconUrl()', () => {
       return false;  
     }
   };
+
+  test('Return contains the davID within it options',() => {     
+    expect(       
+      checkIfReturnDavId(getDavIdIconUrl(dummyDavId))     
+    ).toBeTruthy();   
+  });    
+  
+  const checkIfReturnDavId = (url) => {     
+    if (url.toLowerCase().indexOf(dummyDavId) >= 0) {       
+      return true;     
+    } else {       
+      return false;     
+    }   
+  }
```

Comment:
This line should end with a semicolon.
When running `npm test`, you may have noticed the testing actually failed because of a linting error.

<img width="628" alt="lint" src="https://user-images.githubusercontent.com/926653/33805467-2a406cd6-ddc2-11e7-979c-1fd2fa99578c.png">